docs/tutorials/scala/mxnet_scala_on_intellij.md

```diff
@@ -28,28 +33,30 @@ brew cask install java8
 
 **Step 3.** Install maven:
 ```
-brew update
 brew install maven
 ```
 
-## Build the MXNet Shared Library and Scala Package
+**Step 4.** Install OpenCV:
+```
+brew install opencv@2
```

Comment:
can you also please check other platforms by running on a Ubuntu AWS Instance(not using DL AMI). May be you can make that a separate task when you update the main scala install page.